pygmt/src/subplot.py

```diff
@@ -89,23 +89,24 @@ def subplot(self, nrows=1, ncols=1, **kwargs):
         Reserve a space of dimension *clearance* between the margin and the
         subplot on the specified side, using *side* values from **w**, **e**,
         **s**, or **n**; or **x** for both **w** and **e**; or **y** for both
-        **s** and **n**. No *side* means all sides (i.e. ``clearance='1c'``
+        **s** and **n**. No *side* means all sides (i.e. ``clearance="1c"``
         would set a clearance of 1 cm on all sides). The option is repeatable
-        to set aside space on more than one side (e.g. ``clearance=['w1c',
-        's2c']`` would set a clearance of 1 cm on west side and 2 cm on south
-        side). Such space will be left untouched by the main map plotting but
-        can be accessed by methods that plot scales, bars, text, etc.
+        to set aside space on more than one side (e.g.
+        ``clearance=["w1c","s2c"]`` would set a clearance of 1 cm on west
```

Comment:
I think we should keep the whitespace between items:
```suggestion
        ``clearance=["w1c", "s2c"]`` would set a clearance of 1 cm on west
```

Reply:
I only removed these whitespaces to avoid line breaks. E.g.:

![subplot_doc_spaces_linebreak](https://user-images.githubusercontent.com/94163266/188067939-f75827dc-4ed1-4892-a631-55b51ee41609.png)

In case you think these are no problem, I will remove this change.

Reply:
I don't think these line breaks are problems.

Reply:
Thanks for your fast answer @seisman! Removed the change in https://github.com/GenericMappingTools/pygmt/pull/2104/commits/78688eb9e696fc731a5850a5ee70016195738e34.